test/runtime/kafka.go

```diff
@@ -90,6 +93,18 @@ var _ = Describe("RuntimeValidatedKafka", func() {
 		vm.Exec(cmd)
 	}
 
+	// WaitKafkaBroker waits for the broker to be ready, by executing
+	// a produce request on existing topics and waiting for a response from broker
+	waitForKafkaBroker := func(pod string, cmd string) error {
+		body := func() bool {
+			res := vm.ContainerExec(pod, cmd)
+			fmt.Println(res.CombineOutput())
```

Comment:
Could you delete this Println please? Logs are already in the logs file

Reply:
sure...

Reply:
done